cmd/gitops/beta/run/cmd.go

```diff
@@ -33,6 +53,18 @@ gitops beta run ./deploy/overlays/dev [flags]`,
 		DisableAutoGenTag: true,
 	}
 
+	cmdFlags := cmd.Flags()
+
+	cmdFlags.StringVar(&flags.FluxVersion, "flux-version", "v0.31.3", "The version of Flux to install")
```

Comment:
Could we pull this in from the binary itself? I think we have a good number in `pkg/version` we're not really using at the moment - then we just need to bump the version in `FLUX_VERSION` in the `Makefile` before releasing.

Reply:
Added! But now we need to keep in mind that if we are testing from the source code (like `go run main.go beta run ./`) without running `make gitops` first, Flux will be installed and when running the command again there a message `Flux version "undefined" is found` will be displayed. When running the command from `bin/gitops`, the version is displayed properly.

Reply:
Maybe we can have it default to the string "latest", and github will take care of redirecting us to the most recent release tag automatically :thinking:

Reply:
OK, great! Should we make it default to "latest" only in the command source (with first checking if it's undefined) or should just replace "undefined" with "latest" in `pkg/version`?

Reply:
Just replace "undefined", I think.